TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/ChangeRequests/EditChangeRequest/Reject.cshtml.cs

```diff
@@ -199,13 +203,13 @@ public override async Task OnPageHandlerExecutionAsync(PageHandlerExecutingConte
 
     public enum CaseRejectionReasonOption
     {
-        [Display(Name = "Request and proof don’t match")]
+        [Display(Name = "Request and proof don’t match", Description = "This is because the proof you provided did not match your request.")]
```

Comment:
Can we use a different (custom) attribute for these rejection descriptions? The `Description` property is normally used for hint text on the Console for support agents and this doesn't feel like it's that.